orttraining/orttraining/test/python/orttraining_test_lstm.py

```diff
@@ -840,9 +840,9 @@ def backward_ort(
 
 
 @pytest.mark.parametrize("sequence_length", [2, 4, 16, 32])
-@pytest.mark.parametrize("batch_size", [2, 4, 32])
-@pytest.mark.parametrize("input_size", [2, 4, 32])
-@pytest.mark.parametrize("hidden_size", [2, 4, 32])
+@pytest.mark.parametrize("batch_size", [32])
```

Comment:
I wanted to avoid burdening the CI pipeline with unnecessary tests. Which is why I removed the `[2, 4]` from this parameterized test.
Later I removed the test altogether from being run in the pipeline since we have C++ tests for the expected outputs from the gradient kernel.

I can add back the test to the pipeline if we feel it is needed.